webrender/src/renderer.rs

```diff
@@ -1046,9 +1046,11 @@ impl Renderer {
         let render_target_debug = options.render_target_debug;
         let payload_tx_for_backend = payload_tx.clone();
         let recorder = options.recorder;
+        // TODO(gw): Use a heuristic to select best # of worker threads.
+        let worker_config = rayon::Configuration::new().thread_name(|idx|{ format!("WebRender:Worker#{}", idx) })
+                                                       .num_threads(4);
```

Comment:
let's not even call `num_threads`?

> If num_threads is 0, or you do not call this function, then the Rayon runtime will select the number of threads automatically. At present, this is based on the RAYON_RS_NUM_CPUS (if set), or the number of logical CPUs (otherwise).